apps/web/src/components/templates/VariableManager.tsx

```diff
@@ -0,0 +1,209 @@
+import { IEmailBlock } from '@novu/shared';
```

Comment:
Do you think we can add `.cy.tsx` component tests here to test the edgecases here in isolation?